contents/founders/write-great-job-ads.md

```diff
@@ -0,0 +1,144 @@
+
+However, there are a bunch of things that are actually kinda annoying and unhelpful about it which could put great candidates off applying - see if any stick out to you. We'll come back to this at the end!
+
+![Apple](https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/apple-job-ad.png)
+
+My aim here by the way isn't to say 'PostHog is better at job ads than Apple' - the point is that startups do not have the incredible brand recognition that Apple does, so you need to put in extra effort to stand out!
+
+## Obvious ways to write a good job ad
+
+These are table stakes. If you're not doing these, it's probably not even worth looking at the non-obvious stuff. Yes I am going to mention salary transparency again, like every other nano-influencer on LinkedIn.
+
+**Really tight requirements**
+
+Keep your list of requirements as short as possible. I like to do:
+
+- Must have
+
+- Nice to have
+
+Each should be ~3-4 items long - that's it. You will know anyway when you see a great resume. Candidates don't read this stuff super carefully anyway, or will ignore the stuff they can't do (especially if they are a dude). You need to remember that you are just one of potentially dozens of applications that a candidate is making, so you shouldn't assume that they will read your beautifully crafted ad anywhere near as closely as you did.
+
+Fun fact - you do not need to write out literally every single thing the person will be doing. 
+
+**What the actual hiring process is**
+
+This is nice for the candidate, but more importantly it forces _you_ to figure out how you are going to decide whether a candidate is great or not. A series of random chats with the team is not going to give you the signal you need. 
+
+Even if you are a tiny team, you can still take 15 minutes to figure this out and save yourself a lot of pain improvising the next step and getting stressed you haven't got back to a candidate. 
+
+It's also worth noting that a big advantage startups have over bigger companies is speed - you can potentially go from initial interview to offer in the time it takes a much larger company to even schedule in the first call. Being able to do this gives you a huge advantage, but it's impossible unless you nail down your process first.
+
+(And by the way, if you follow your process and are still not sure if you have enough signal about whether a candidate is great, it probably means that you want to say 'no'!)
+
+**Time zones**
+
+All-remote doesn't necessarily mean hiring from literally anywhere. Please specify time zones before people waste a bunch of time, only to find out you don't hire in Australia. This also forces you to clarify your own thinking.
+
+**Salary**
+
+Stick a range in there if you really don't know. You may live somewhere this is legally mandated anyway. The point here is to avoid wasting a ton of your _own_ time getting someone through your process, only to find you are miles apart here. 
```

Comment:
```suggestion
Stick a range in there if you really don't know. You may be based somewhere that this is legally mandated anyway. The point here is to avoid wasting a ton of your _own_ time getting someone through your process, only to find you are miles apart on how much the role pays. 
```